core/src/main/java/io/grpc/Metadata.java

```diff
@@ -373,34 +378,6 @@ public String toString() {
   }
 
   /**
-   * Concrete instance for metadata attached to the end of the call. Only provided by
-   * servers.
-   */
-  public static class Trailers extends Metadata {
```

Comment:
Yeah. We could remove the byte[] constructor and the toString() without issue, but the public constructor is the main thing that existing code would be using.